containers/unit-tests/run.sh

```diff
@@ -32,7 +32,8 @@ case $ARCH in
 esac
 
 if [ -d bots ]; then
-    bots/test-bots
+    # Set GITHUB_BASE so that "import task" works without failure.
+    GITHUB_BASE=unused bots/test-bots
```

Comment:
FWIW, it seems a little strange to set this to this magic value, instead of  `env -u GITHUB_BASE`. But fine for me as long as it works.

Reply:
Yeah, we should link to a bug report.  I'll file one.